README.md

```diff
@@ -27,7 +27,7 @@ defmodule HTTPSandbox do
   @registry :http_sandbox
   @keys :unique
   # state is a sub-key to allow multiple contexts to use the same registry
-  @state "http"
+  @state %{state: "http"}
```

Comment:
Is this correct? I think that this might be the context

Reply:
Yes, `state` is a map. context is an atom:
![image](https://github.com/alanvardy/sandbox_registry/assets/51061538/a6016ae0-8652-4d9b-8332-616389413cee)